javascriptv3/example_code/libs/scenario/scenario-example.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Scenario,
+  ScenarioAction,
+  ScenarioInput,
+  ScenarioOutput,
+} from "./index.js";
+
+const greet = new ScenarioOutput(
+  "greet",
+  "Hi! This is a scenario. It can handle your " + "inputs and outputs.",
```

Comment:
Why the concat?

Reply:
It was multiple lines before prettier got to it. I'll fix it.